### PR TITLE
updated folder structure and readme's

### PR DESCRIPTION
**Pull request by pmgrove**
:v:

/cc @zendesk/vegemite

### Description

Updated folder structure to support v2 demo apps for Zendesk Support and Chat.

### References
* JIRA: 

### Risks
* links in other documentation to v1 demo apps may break

### CODE REVIEW

**Comment by sandlerr** [comment]
Good thing somebody is doing some work around here

screen shot 2016-10-13 at 11 52 47 am

File: README.md [hunk 1]
```diff
@@ -5,177 +5,3 @@
 This repository provides you with sample Apps to help learn how to use the Zendesk Apps framework and APIs.
 
 We hope you'll find those sample Apps useful and encourage you to re-use some of this code in your own Apps. If you have any questions please email support@zendesk.com or find us on [Twitter](https://twitter.com/zendeskdevteam). Please submit bug reports to [Zendesk](https://support.zendesk.com/requests/new). Pull requests are welcome.
-
```
Comment (adammw) [inline README.md]: Links to v1 and v2 folders?

**Comment by pmgrove** [comment]
@adammw 💇

**Comment by adammw** [comment]
👍

**Pull request merged**